source/PlayerInfo.cpp

```diff
@@ -1632,6 +1617,46 @@ bool PlayerInfo::TakeOff(UI *ui)
 
 
 
+void PlayerInfo::PoolCargo()
+{
+	// This can only be done while landed.
+	if(!system || !planet)
```

Comment:
```suggestion
	if(!planet)
```
why check for the system?